src/symposium-homepage.html

```diff
@@ -1,5 +1,4 @@
 <link rel="import" async href="../bower_components/google-map/google-map.html">
-<link rel="import" async href="../bower_components/google-map/google-map-marker.html">
```

Comment:
Because it is lazy-loaded here: https://github.com/GoogleWebComponents/google-map/compare/master...TimvdLippe:intersection-observer#diff-1de9a590f21defa41d706b424e83bb63R458